devnet/substrate_alice.Dockerfile

```diff
@@ -16,9 +16,6 @@ RUN go run cmd/update-dd-agent-confd/main.go -n=${METRICS_NAMESPACE} -t=key:alic
 
 FROM parity/polkadot:${POLKADOT_VERSION}
 
-ARG POLKADOT_VERSION
-# Using a genesis file with 3 authority nodes (alice, bob, charlie) generated using polkadot $POLKADOT_VERSION
-ARG CHAIN=3-auth-node-${POLKADOT_VERSION}
```

Comment:
Same here, could we keep the `ARG CHAIN=westend-local` and use it in the `ENTRYPOINT` command?